gpt_engineer/preprompts/ht.txt

```diff
@@ -0,0 +1,2 @@
+The improve prompt is strongly based on https://github.com/paul-gauthier/aider/blob/main/aider/coders/editblock_coder.py
+h/t Paul Gauthier
```

Comment:
Better to add this to acknowledgements.md:)

Reply:
or what does ht stand for?

Reply:
Created `acknowledgements.md` & deleted this - feel free to adjust as you like later!

h/t stands for "hat tip" - see https://slang.net/meaning/h/t.